t/mldistwatch-perl.t

```diff
@@ -96,22 +96,67 @@ subtest "indexing a new perl" => sub {
   die "couldn't make OPRIME a pumpking"
     unless $dbh->do("INSERT INTO grouptable (user, ugroup) VALUES ('OPRIME', 'pumpking')");
 
-  $pause->upload_author_fake(OPRIME => {
-    name      => 'perl',
-    version   => '5.56.55',
-    packages  => [ 'Perl::Core' ],
-    packages  => [
-      'Perl::Core' => { version => '1.002' },
-    ],
-  });
-
-  my $result = $pause->test_reindex;
-
-  $result->package_list_ok(
-    [
-      { package => 'Perl::Core',      version => '1.002' },
-    ],
-  );
+  subtest "first version of perl" => sub {
+    $pause->upload_author_fake(OPRIME => {
+      name      => 'perl',
+      version   => '5.56.55',
+      packages  => [
+        'Perl::Core'    => { version => '1.002' },
+        'Little::Buddy' => { version => '2.003' },
+      ],
+    });
+
+    my $result = $pause->test_reindex;
```

Comment:
There are a bunch of places we don't do this but should.  I will probably add them later, but I'm not going to make an issue for it now, because I think it's very low priority.